manifests/f/FastStone/Viewer/7.6/FastStone.Viewer.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: FastStone.Viewer
+PackageVersion: "7.6"
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.1.1
```

```suggestion
ManifestVersion: 1.1.0
```
`1.1.1` schema doesn't currently exist.